qcportal/qcportal/dataset_models.py

```diff
@@ -1118,27 +1116,130 @@ def uninvalidate_records(
     def compile_values(
         self,
         value_call: Callable,
-        value_name: str,
+        value_names: Union[List[str], str] = "value",
         entry_names: Optional[Union[str, Iterable[str]]] = None,
         specification_names: Optional[Union[str, Iterable[str]]] = None,
-        include: Optional[Iterable[str]] = None,
-        fetch_updated: bool = True,
-        force_refetch: bool = False,
+        unpack: bool = False,
     ) -> pd.DataFrame:
-        def _data_generator():
+        """
+        Compile values from records into a pandas DataFrame.
+
+        Parameters
+        -----------
+        value_call : Callable
```

Comment:
We don't use the types in the docstrings, since sphinx can now get the types from the function arguments itself.